tools/common/StaticRegistrar.cs

```diff
@@ -2529,8 +2529,13 @@ void Specialize (AutoIndentStringBuilder sb)
 				if (@class.Methods == null && isPlatformType && !@class.IsProtocol && !@class.IsCategory)
 					continue;
 
-				if (@class.IsModel)
-					continue;
+#if MMP
+				switch (@class.ExportedName) {
+				case "AVCaptureDataOutputSynchronizer":
+				case "AVCaptureDataOutputSynchronizerDelegate":
+					continue; // We've accidentally added bindings for these two classes which are not available in macOS.
+				}
```

Comment:
Can we just stub them on macOS ? i.e. they don't need the attributes or export/selectors that would make the static registrar triggers on them ?